### ISSUE TITLE
Fails weirdly if the aports directory is not named 'aports'

### ISSUE THREAD

**Issue by z3ntu** [alpinelinux/docker-abuild#38]
I had the folder called ´aports_local` and running dabuild failed weirdly with this error:
```
>>> ERROR: : Could not find ./APKBUILD (PWD=/home/builder/aports_local/community/brotli)
```
After renaming the folder from aports_local to aports, it works fine.

**Comment by mor1** [comment]
Yes, the script assumes your aports directory is called `aports/`.
If you can give me more detail about your configuration, I can try to recreate, and perhaps deal with it better.

**Comment by z3ntu** [comment]
My aports folder was called `aports_local`, I'm guessing that's how you can reproduce the issue yourself. From looking at the message

> Error: expecting to be run from within an aports tree! Could not find '/aports' in the current path (/home/luca)

I'm guessing the script checks if literally `/aports` is in the path name which it is if the folder is called `aports_local`

**Comment by mor1** [comment]
Right-- that's why I put that error message in there. The script does indeed check that explicitly-- https://github.com/alpinelinux/docker-abuild/blob/master/dabuild.in#L19-L24.

The script was supposed to have bailed out at that point though, so I'm a bit surprised you got the `APKBUILD` error you quoted at the top.

Do you have a particular need to call your `aports` tree something else? Would it help if, eg., I allowed you to indicate the name of your `aports` tree as an environment variable?

**Comment by z3ntu** [comment]
No, I just had it called like that because the other folder called "aports" which I used to mount over sshfs. So there are two solutions to this I think: fix the code to accept names other than `aports` or fix the check so it only accepts `aports` and not e.g. `aports2`

**Comment by mor1** [comment]
Ah! I see -- yes, that makes sense. Accepting only `aports` would certainly be simpler -- I'll do that. Thanks!

**Comment by mor1** [comment]
Will be fixed once #39 is merged; closing
